{
  "parameter(s)": [
    537
  ],
  "test_sublist_size": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00022
  ],
  "test_list_of_several_same_number": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00137
  ],
  "test_list_of_single_number": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00026
  ]
}